MaxText/configs/base.yml

```diff
@@ -40,8 +40,20 @@ gcs_metrics: False
 # Activation dtypes.
 dtype: "bfloat16"
 int8_training: False
+# The following flags will only have effect when int8_training is True
+use_dqdg: False
 fwd_int8: True
-bwd_int8: True
+dlhs_int8: True
+drhs_int8: False
+fwd_int8_qk: False
+dlhs_int8_qk: False
+drhs_int8_qk: False
+fwd_int8_pv: True
+dlhs_int8_pv: True
+drhs_int8_pv: False
+aqt_use_dummy_static_bound: False
+aqt_use_fwd_quant: False
+aqt_rng_type: "jax.uniform"
 
 # Global parameter scale needs to be a power of 2. If you want finer grained control of the model sizes 
```

Comment:
These parameters were different in our int8 runs:
- remat_policy
- per_device_batch_size
- max_target_length
- enable_dropout

Also, we have `config.global_batch_size_to_load` now.

What should we do about it?

Reply:
I don't see the `global_batch_size_to_load` parameter you're referring to in int8-sweeps base.yml, what are you referring to?

Reply:
Remat policy: Our best medium/large configs do use remat policy of full, and since int8 works much better with remat full I think we should change to the default to full, wdyt @rwitten?

per_device_batch_size: This is expected to change with each model, the default here is optimized for the default size of 1B. Users should expect (or will quickly learn) to decrease it as the model size increases. We publish optimized configs for larger models with chosen batch sizes in MaxText/configs, e.g. [64b.sh](https://github.com/google/maxtext/blob/main/MaxText/configs/64b.sh) which uses this [config](https://github.com/google/maxtext/blob/main/end_to_end/test_tflops_64b_params.sh)

max_target_length: This was changed when flash attention was added, I believe we should keep it at 2k moving forward. Again we publish optimized configs of various sizes where we explicitly set this parameter.

enable_dropout: I changed this to false for int8 so we had deterministic runs, but the dropout_rate is zero by default as well so its also deterministic when true until you change the dropout_rate. I don't think the default value is particularly important here

Reply:
Yes lets move everything to full remat.